### ISSUE TITLE
EM Scatter tutorial hangs at AffineFEOperator(a,b,U,V)

### ISSUE THREAD

**Issue by AndiMD** [gridap/Tutorials#118]
The EM Scatter tutorial hangs at 100%CPU with Julia 1.7.0-beta4, Gridap v0.16.5.

[https://gridap.github.io/Tutorials/stable/pages/t012_emscatter/#Tutorial-12:-Electromagnetic-scattering-in-2D-1](https://gridap.github.io/Tutorials/stable/pages/t012_emscatter/#Tutorial-12:-Electromagnetic-scattering-in-2D-1)

After 2h+, I cancelled the computation
```op = AffineFEOperator(a,b,U,V)```
```
^C^C^C^C^C^C^C^CWARNING: Force throwing a SIGINT
Internal error: encountered unexpected error in runtime:
InterruptException()
segv_handler at /buildworker/worker/package_linux64/build/src/signals-unix.c:310
^C^C^C^C^C^C^C^C^Cc^C/usr/local/bin/julia: line 3:  5021 Segmentation fault      (core dumped)
```

**Comment by fverdugo** [comment]
Hi @AndiMD, is only this tutorial?

**Comment by AndiMD** [comment]
> Hi @AndiMD, is only this tutorial?

yes, it is only this one. Running in the instantiated Tutorials environment.
